vello_tests/src/snapshot.rs

```diff
@@ -147,14 +147,24 @@ pub fn smoke_snapshot_test_sync(scene: Scene, params: &TestParams) -> Result<Sna
     pollster::block_on(snapshot_test(scene, params, SnapshotDirectory::Smoke))
 }
 
-/// Run a snapshot test of the given scene.
```

Comment:
Why remove this?

Reply:
I honestly have no idea :upside_down_face:

My original plan in this PR was to downscale the image for the test, but I realised just directly testing the behaviour would be better.
(Fwiw, once #680 is fixed, then downscaled tests is *probably* the right approach there)